annotator/annotator.py

```diff
@@ -356,10 +392,13 @@ def __init__(self, start, end, doc, label=None):
 
     def overlaps(self, other_span):
         return (
-            (self.start >= other_span.start and self.start <= other_span.end) or
-            (other_span.start >= self.start and other_span.start <= self.end)
+            (self.start >= other_span.start and self.start < other_span.end) or
+            (other_span.start >= self.start and other_span.start < self.end)
```

Comment:
self.start = 10, self.end=12
other.start = 5, other.end = 10

These overlap but won't be caught anymore, is this ok?